x509/root_darwin_armx.go

```diff
@@ -4,6 +4,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:build cgo && darwin && (arm || arm64 || ios)
```

Comment:
Was this auto-generated by `gofmt`?

Reply:
Yes, it was auto-generated when I executed `go fmt ./...`.

Reply:
@pavelkalinnikov Gentle ping.